docs/src/modules/components/DemoSandboxed.js

```diff
@@ -61,17 +61,29 @@ function DemoFrame(props) {
    */
   const frameRef = React.useRef(null);
 
-  const [mounted, setMounted] = React.useState(false);
+  // If we portal content into the iframe before the load event then that content
+  // is dropped in firefox.
+  const [iframeLoaded, onLoad] = React.useReducer(() => true, false);
+
   React.useEffect(() => {
-    setMounted(true);
-  }, []);
+    const document = frameRef.current.contentDocument;
+    // When we hydarte the iframe then the load event is already dispatched
+    // once the iframe markup is parsed (maybe later but the important part is
+    // that it happens before React can attach event listeners).
+    // We need to check the readyState of the document once the iframe is mounted
+    // and "replay" the missed load event.
+    // See https://github.com/facebook/react/pull/13862 for ongoing effort in React
+    // (though not with iframes in mind).
+    if (document != null && document.readyState === 'complete' && !iframeLoaded) {
```

Comment:
Feel free to ignore this particular suggestion but as a rule of thumb I try to always derive `document` from `ownerDocument`. Plus I haven’t actually tested this code, who knows, maybe iframes don’t even have an `ownerDocument` property 😅

```suggestion
    const doc = frameRef.current.ownerDocument;
    if (doc != null && doc.readyState === 'complete' && !iframeLoaded) {
```